Formula/t/touca.rb

```diff
@@ -12,6 +13,8 @@ class Touca < Formula
     sha256 cellar: :any_skip_relocation, all: "28347dcea2a95cf714fad75fb909df2202c64a2ba9f9b83af3f99a7659e0bf17"
   end
 
+  deprecate! date: "2025-08-13", because: :unmaintained
```

Comment:
Can we file an issue upstream asking when they plan to cut a new release before deprecating?

Reply:
Considering the install numbers, a lack of commits since early 2024, and the fact that the main person working on the project now states on their GitHub they are no longer with the company, I don't think that's necessary.